tinytuya/scanner.py

```diff
@@ -208,32 +208,38 @@ def send_discovery_request( iface_list=None ):
         client_bcast_addrs = get_ip_to_broadcast()
         for bcast in client_bcast_addrs:
             addr = client_bcast_addrs[bcast]
-            iface_list[addr] = { 'broadcast': bcast }
+            iface_list[addr] = {'broadcast': bcast}
 
     for address in iface_list:
         iface = iface_list[address]
         if 'socket' not in iface:
-            iface['socket'] = socket.socket(socket.AF_INET, socket.SOCK_DGRAM) # UDP
-            iface['socket'].setsockopt(socket.SOL_SOCKET, socket.SO_BROADCAST, 1)
-            iface['socket'].bind( (address,0) )
+            try:
+                print(f"Attempting to bind to address: {address}")
+                iface['socket'] = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)  # UDP
+                iface['socket'].setsockopt(socket.SOL_SOCKET, socket.SO_BROADCAST, 1)
+                iface['socket'].bind((address, 0))
+            except socket.gaierror as e:
+                print(f"Failed to bind to {address}: {e}")
+                continue
```

Comment:
Thanks for this PR @blitzu !  I'm wondering, why use `print()` instead of `log.debug()` and `log.error()` here?

Reply:
this is what chatgpt gave me.  i had an error on using scanner.py and this was the solution from chatgpt and it works